code/game/objects/structures/girders.dm

```diff
@@ -242,7 +242,7 @@
 			P.loc = src.loc
 			user << "<span class='notice'>You fit the pipe into \the [src].</span>"
 	else
-		return ..()
+		..()
```

Comment:
Hmm wonder why it got changed, I only touched this file to change the decon material to metal